asreview/webapp/tests/data/asreview-project-v0-19-startreview/project.json

```diff
@@ -0,0 +1 @@
+{"version": "0.19", "id": "asreview-project-v0-19-startreview", "name": "ASReview Project v0.19 startReview", "description": "", "authors": "", "created_at_unix": 1648717742, "projectInitReady": true, "reviewFinished": false, "dataset_path": "Kwok_2020.csv"}
```

Comment:
Is this a replacement for the use of `asreview/asreview-project-files-testing`? I'm not a big fan of having large files in the repo (I try to remove them). Would a git submodule be useful?

Reply:
Yeah I think so, lemme have a look.

Reply:
Fixed with downloading projects from Github into the asreview test folder. Open for enhancement.